hudi-utilities/src/test/java/org/apache/hudi/utilities/offlinejob/TestHoodieCompactorJob.java

```diff
@@ -59,7 +59,6 @@ public void testHoodieCompactorWithOptionalClean(boolean skipClean) throws Excep
         .withSchema(TRIP_EXAMPLE_SCHEMA)
         .withParallelism(2, 2)
         .withMetadataConfig(HoodieMetadataConfig.newBuilder().enable(false).build())
-        .withAutoCommit(false)
```

Comment:
nit: Check if auto commit disabled (compared to previous enabled as default) has any implication in tests.

Reply:
https://issues.apache.org/jira/browse/HUDI-9440